# Chinese (Taiwan) translations for keystone.
# Copyright (C) 2013 ORGANIZATION
# This file is distributed under the same license as the keystone project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version:  Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2013-06-14 17:05+0000\n"
"PO-Revision-Date: 2013-05-22 03:11+0000\n"
"Last-Translator: daisy.ycguo <daisy.ycguo@gmail.com>\n"
"Language-Team: Chinese (Taiwan) "
"(http://www.transifex.com/projects/p/openstack/language/zh_TW/)\n"
"Plural-Forms: nplurals=1; plural=0\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"

#: keystone/clean.py:23
#, python-format
msgid "%s cannot be empty."
msgstr "%s 不能是空的。"

#: keystone/clean.py:25
#, python-format
msgid "%(property_name)s cannot be less than %(min_length)s characters."
msgstr "%(property_name)s 不能少於 %(min_length)s 個字元。"

#: keystone/clean.py:29
#, python-format
msgid "%(property_name)s should not be greater than %(max_length)s characters."
msgstr "%(property_name)s 不應超過 %(max_length)s 個字元。"

#: keystone/clean.py:36
#, python-format
msgid "%(property_name)s is not a %(display_expected_type)s"
msgstr ""

#: keystone/test.py:117
#, python-format
msgid "Failed to checkout %s"
msgstr "無法移出 %s"

#: keystone/auth/controllers.py:72
#, python-format
msgid "Project is disabled: %s"
msgstr "已停用專案：%s"

#: keystone/auth/controllers.py:78 keystone/auth/plugins/password.py:39
#, python-format
msgid "Domain is disabled: %s"
msgstr "已停用網域：%s"

#: keystone/auth/controllers.py:84 keystone/auth/plugins/password.py:45
#, python-format
msgid "User is disabled: %s"
msgstr "已停用使用者：%s"

#: keystone/auth/controllers.py:265
msgid "Scoping to both domain and project is not allowed"
msgstr "不容許將範圍同時設定為網域及專案"

#: keystone/auth/controllers.py:268
msgid "Scoping to both domain and trust is not allowed"
msgstr "不容許將範圍同時設定為網域及信任"

#: keystone/auth/controllers.py:271
msgid "Scoping to both project and trust is not allowed"
msgstr "不容許將範圍同時設定為專案及信任"

#: keystone/auth/controllers.py:333
#, python-format
msgid "Unable to lookup user %s"
msgstr "無法查閱使用者 %s"

#: keystone/auth/controllers.py:363
msgid "User not found"
msgstr "找不到使用者"

#: keystone/auth/token_factory.py:81
msgid "User have no access to project"
msgstr "使用者無法存取專案"

#: keystone/auth/token_factory.py:96
msgid "User have no access to domain"
msgstr "使用者無法存取網域"

#: keystone/auth/token_factory.py:314 keystone/token/controllers.py:121
msgid "Unable to sign token."
msgstr "無法簽署記號。"

#: keystone/auth/token_factory.py:317 keystone/token/controllers.py:124
#, python-format
msgid "Invalid value for token_format: %s.  Allowed values are PKI or UUID."
msgstr "token_format 的值無效：%s。接受的值為 PKI 或 UUID。"

#: keystone/catalog/core.py:38
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr "端點 %(url)s 的格式不正確 - 不明的索引鍵 %(keyerror)s"

#: keystone/catalog/core.py:43
#, python-format
msgid ""
"Malformed endpoint %(url)s - unknown key %(keyerror)s(are you missing "
"brackets ?)"
msgstr "端點 %(url)s 的格式不正確 - 不明的索引鍵 %(keyerror)s（遺漏了方括弧嗎？）"

#: keystone/catalog/core.py:49
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format                   (are you "
"missing a type notifier ?)"
msgstr "端點 %s 的格式不正確 - 格式不完整（遺漏了類型通知符嗎？）"

#: keystone/catalog/backends/templated.py:109
#, python-format
msgid "Unable to open template file %s"
msgstr "無法開啟範本檔 %s"

#: keystone/common/cms.py:42
#, python-format
msgid "Verify error: %s"
msgstr "驗證發生錯誤：%s"

#: keystone/common/cms.py:134
msgid ""
"Signing error: Unable to load certificate - ensure you've configured PKI "
"with 'keystone-manage pki_setup'"
msgstr "簽署發生錯誤：無法載入憑證 - 請確保已使用 'keystone-manage pki_setup' 來配置 PKI"

#: keystone/common/cms.py:138
#, python-format
msgid "Signing error: %s"
msgstr "簽署發生錯誤：%s"

#: keystone/common/config.py:89
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr "找不到指定的記載配置檔：%s"

#: keystone/common/config.py:107
msgid "Invalid syslog facility"
msgstr "無效的 Syslog 機能"

#: keystone/common/controller.py:18
#, python-format
msgid "RBAC: Authorizing %(action)s(%(kwargs)s)"
msgstr ""

#: keystone/common/controller.py:26
msgid "RBAC: Invalid token"
msgstr "RBAC：無效的記號"

#: keystone/common/controller.py:36 keystone/common/controller.py:57
msgid "RBAC: Invalid user"
msgstr "RBAC：無效的使用者"

#: keystone/common/controller.py:42
msgid "RBAC: Proceeding without project"
msgstr "RBAC：在沒有專案的情況下繼續作業"

#: keystone/common/controller.py:62
msgid "RBAC: Proceeding without tenant"
msgstr "RBAC：在沒有 Tenant 的情況下繼續作業"

#: keystone/common/controller.py:92 keystone/common/controller.py:144
msgid "RBAC: Bypassing authorization"
msgstr "RBAC：正在略過授權"

#: keystone/common/controller.py:101 keystone/common/controller.py:142
msgid "RBAC: Authorization granted"
msgstr "RBAC：已授與權限"

#: keystone/common/controller.py:131
#, python-format
msgid "RBAC: Adding query filter params (%s)"
msgstr "RBAC：正在新增查詢過濾器參數 (%s)"

#: keystone/common/controller.py:322
msgid "Invalid token in normalize_domain_id"
msgstr "normalize_domain_id 中的記號無效"

#: keystone/common/utils.py:232
msgid ""
"Error setting up the debug environment. Verify that the option --debug-"
"url has the format <host>:<port> and that a debugger processes is "
"listening on that port."
msgstr ""

#: keystone/common/wsgi.py:162
#, python-format
msgid "arg_dict: %s"
msgstr "arg_dict：%s"

#: keystone/common/wsgi.py:186
#, fuzzy, python-format
msgid "Authorization failed. %(exception)s from %(remote_addr)s"
msgstr "授權失敗。%s（自 %s）"

#: keystone/common/wsgi.py:429
msgid "The resource could not be found."
msgstr "找不到資源。"

#: keystone/common/wsgi_server.py:72
#, python-format
msgid "Starting %(arg0)s on %(host)s:%(port)s"
msgstr "正在 %(host)s:%(port)s 上啟動 %(arg0)s"

#: keystone/common/wsgi_server.py:132
msgid "Server error"
msgstr "伺服器錯誤"

#: keystone/common/ldap/core.py:79
#, python-format
msgid "Invalid LDAP deref option: %s. Choose one of: "
msgstr "無效的 LDAP deref 選項：%s。請選擇下列其中一個："

#: keystone/common/ldap/core.py:87
#, fuzzy, python-format
msgid "Invalid LDAP TLS certs option: %(option). Choose one of: %(options)s"
msgstr "無效的 LDAP deref 選項：%s。請選擇下列其中一個："

#: keystone/common/ldap/core.py:99
#, fuzzy, python-format
msgid "Invalid LDAP scope: %(scope)s. Choose one of: %(options)s"
msgstr "無效的 LDAP 範圍：%s。請選擇下列其中一個："

#: keystone/common/ldap/core.py:189
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%s\". Format must be "
"<ldap_attribute>:<keystone_attribute>"
msgstr ""

#: keystone/common/ldap/core.py:195
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%(item)s\". Value "
"\"%(attr_map)s\" must use one of %(keys)s."
msgstr ""

#: keystone/common/ldap/core.py:279 keystone/identity/backends/kvs.py:596
#: keystone/identity/backends/kvs.py:624
#, python-format
msgid "Duplicate name, %s."
msgstr "重複的名稱，%s。"

#: keystone/common/ldap/core.py:289 keystone/identity/backends/kvs.py:589
#, python-format
msgid "Duplicate ID, %s."
msgstr "重複的 ID，%s。"

#: keystone/common/ldap/core.py:294
#, python-format
msgid "LDAP %s create"
msgstr "LDAP %s 建立"

#: keystone/common/ldap/core.py:372
#, python-format
msgid "LDAP %s update"
msgstr "LDAP %s 更新"

#: keystone/common/ldap/core.py:405
#, python-format
msgid "LDAP %s delete"
msgstr "LDAP %s 刪除"

#: keystone/common/ldap/core.py:430
#, python-format
msgid "LDAP init: url=%s"
msgstr "LDAP 起始設定：URL = %s"

#: keystone/common/ldap/core.py:431
#, python-format
msgid ""
"LDAP init: use_tls=%(use_tls)s\n"
"tls_cacertfile=%(tls_cacertfile)s\n"
"tls_cacertdir=%(tls_cacertdir)s\n"
"tls_req_cert=%(tls_req_cert)s\n"
"tls_avail=%(tls_avail)s\n"
msgstr ""

#: keystone/common/ldap/core.py:450
msgid "Invalid TLS / LDAPS combination"
msgstr ""

#: keystone/common/ldap/core.py:454
#, python-format
msgid "Invalid LDAP TLS_AVAIL option: %s. TLS not available"
msgstr ""

#: keystone/common/ldap/core.py:464
#, python-format
msgid "tls_cacertfile %s not found or is not a file"
msgstr ""

#: keystone/common/ldap/core.py:476
#, python-format
msgid "tls_cacertdir %s not found or is not a directory"
msgstr ""

#: keystone/common/ldap/core.py:483
#, python-format
msgid "LDAP TLS: invalid TLS_REQUIRE_CERT Option=%s"
msgstr ""

#: keystone/common/ldap/core.py:497
#, python-format
msgid "LDAP bind: dn=%s"
msgstr "LDAP 連結：DN = %s"

#: keystone/common/ldap/core.py:508
#, fuzzy, python-format
msgid "LDAP add: dn=%(dn)s, attrs=%(attrs)s"
msgstr "LDAP 新增：DN = %s，屬性 = %s"

#: keystone/common/ldap/core.py:514
#, fuzzy, python-format
msgid ""
"LDAP search: dn=%(dn)s, scope=%(scope)s, query=%(query)s, "
"attrs=%(attrlist)s"
msgstr "LDAP 搜尋：DN = %s，範圍 = %s，查詢 = %s，屬性 = %s"

#: keystone/common/ldap/core.py:567
msgid ""
"LDAP Server does not support paging. Disable paging in keystone.conf to "
"avoid this message."
msgstr "「LDAP 伺服器」不支援分頁。請在 keystone.conf 中停用分頁以避免此訊息。"

#: keystone/common/ldap/core.py:584
#, fuzzy, python-format
msgid "LDAP modify: dn=%(dn)s, modlist=%(modlist)s"
msgstr "LDAP 修改：DN = %s，Modlist = %s"

#: keystone/common/ldap/core.py:590
#, python-format
msgid "LDAP delete: dn=%s"
msgstr "LDAP 刪除：DN = %s"

#: keystone/common/ldap/core.py:595
#, fuzzy, python-format
msgid "LDAP delete_ext: dn=%(dn)s, serverctrls=%(serverctrls)s"
msgstr "LDAP delete_ext：DN = %s，Serverctrls = %s"

#: keystone/common/ldap/fakeldap.py:146
#, python-format
msgid "FakeLdap initialize url=%s"
msgstr "FakeLdap 起始設定 URL = %s"

#: keystone/common/ldap/fakeldap.py:156
#, python-format
msgid "FakeLdap bind dn=%s"
msgstr "FakeLdap 連結 DN = %s"

#: keystone/common/ldap/fakeldap.py:163
#, python-format
msgid "FakeLdap bind fail: dn=%s not found"
msgstr "FakeLdap 連結失敗：找不到 DN = %s"

#: keystone/common/ldap/fakeldap.py:170
#, python-format
msgid "FakeLdap bind fail: password for dn=%s not found"
msgstr "FakeLdap 連結失敗：找不到 DN = %s 的密碼"

#: keystone/common/ldap/fakeldap.py:175
#, python-format
msgid "FakeLdap bind fail: password for dn=%s does not match"
msgstr "FakeLdap 連結失敗：DN = %s 的密碼不符"

#: keystone/common/ldap/fakeldap.py:190
#, fuzzy, python-format
msgid "FakeLdap add item: dn=%(dn)s, attrs=%(attrs)s"
msgstr "FakeLdap 新增項目：DN = %s，屬性 = %s"

#: keystone/common/ldap/fakeldap.py:193
#, python-format
msgid "FakeLdap add item failed: dn=%s is already in store."
msgstr "FakeLdap 新增項目失敗：DN = %s 已在儲存庫中。"

#: keystone/common/ldap/fakeldap.py:207 keystone/common/ldap/fakeldap.py:221
#, python-format
msgid "FakeLdap delete item: dn=%s"
msgstr "FakeLdap 刪除項目：DN = %s"

#: keystone/common/ldap/fakeldap.py:211 keystone/common/ldap/fakeldap.py:225
#, python-format
msgid "FakeLdap delete item failed: dn=%s not found."
msgstr "FakeLdap 刪除項目失敗：找不到 DN = %s。"

#: keystone/common/ldap/fakeldap.py:240
#, fuzzy, python-format
msgid "FakeLdap modify item: dn=%(dn)s attrs=%(attrs)s"
msgstr "FakeLdap 修改項目：DN = %s 屬性 = %s"

#: keystone/common/ldap/fakeldap.py:245
#, python-format
msgid "FakeLdap modify item failed: dn=%s not found."
msgstr "FakeLdap 修改項目失敗：找不到 DN = %s。"

#: keystone/common/ldap/fakeldap.py:262
#, python-format
msgid "FakeLdap modify item failed: item has no attribute \"%s\" to delete"
msgstr "FakeLdap 修改項目失敗：項目沒有要刪除的屬性 \"%s\""

#: keystone/common/ldap/fakeldap.py:273
#, fuzzy, python-format
msgid ""
"FakeLdap modify item failed: item has no attribute \"%(k)s\" with value "
"\"%(v)s\" to delete"
msgstr "FakeLdap 修改項目失敗：項目沒有要刪除的屬性 \"%s\"（值為 \"%s\"）"

#: keystone/common/ldap/fakeldap.py:279
#, python-format
msgid "FakeLdap modify item failed: unknown command %s"
msgstr "FakeLdap 修改項目失敗：不明的指令 %s"

#: keystone/common/ldap/fakeldap.py:281
#, python-format
msgid "modify_s action %s not implemented"
msgstr "未實作 modify_s 動作 %s"

#: keystone/common/ldap/fakeldap.py:300
#, fuzzy, python-format
msgid "FakeLdap search at dn=%(dn)s scope=%(scope)s query=%(query)s"
msgstr "FakeLdap 搜尋（DN = %s 範圍 = %s 查詢 = %s）"

#: keystone/common/ldap/fakeldap.py:306
msgid "FakeLdap search fail: dn not found for SCOPE_BASE"
msgstr "FakeLdap 搜尋失敗：找不到 SCOPE_BASE 的 DN"

#: keystone/common/ldap/fakeldap.py:320
#, python-format
msgid "Search scope %s not implemented."
msgstr "未實作搜尋範圍 %s。"

#: keystone/common/sql/core.py:205
#, python-format
msgid "Got mysql server has gone away: %s"
msgstr "已取得 mysql 伺服器已斷線的訊息：%s"

#: keystone/common/sql/legacy.py:180
#, python-format
msgid "Cannot migrate EC2 credential: %s"
msgstr "無法移轉 EC2 認證：%s"

#: keystone/common/sql/migration.py:47
msgid "version should be an integer"
msgstr "版本應該是整數"

#: keystone/common/sql/nova.py:62
#, python-format
msgid "Create tenant %s"
msgstr "建立 Tenant %s"

#: keystone/common/sql/nova.py:79
#, python-format
msgid "Create user %s"
msgstr "建立使用者 %s"

#: keystone/common/sql/nova.py:88
#, fuzzy, python-format
msgid "Add user %(user_id)s to tenant %(tenant_id)s"
msgstr "將使用者 %s 新增至 Tenant %s"

#: keystone/common/sql/nova.py:97
#, python-format
msgid "Ignoring existing role %s"
msgstr "正在忽略現有角色 %s"

#: keystone/common/sql/nova.py:104
#, python-format
msgid "Create role %s"
msgstr "建立角色 %s"

#: keystone/common/sql/nova.py:114
#, fuzzy, python-format
msgid "Assign role %(role_id)s to user %(user_id)s on tenant %(tenant_id)s"
msgstr "將角色 %s 指派給使用者 %s（在 Tenant %s 上）"

#: keystone/common/sql/nova.py:133
#, fuzzy, python-format
msgid "Creating ec2 cred for user %(user_id)s and tenant %(tenant_id)s"
msgstr "正在給使用者 %s 及 Tenant %s 建立 EC2 Cred"

#: keystone/identity/backends/kvs.py:250 keystone/identity/backends/kvs.py:259
msgid "User not found in group"
msgstr "在群組中找不到使用者"

#: keystone/identity/backends/sql.py:425
#, python-format
msgid "Cannot remove role that has not been granted, %s"
msgstr "無法移除尚未授權的角色，%s"

#: keystone/identity/backends/ldap/core.py:82
#, python-format
msgid "Expected dict or list: %s"
msgstr ""

#: keystone/identity/backends/ldap/core.py:677
#, python-format
msgid "Role %s not found"
msgstr "找不到角色 %s"

#: keystone/identity/backends/ldap/core.py:894
msgid "Changing Name not supported by LDAP"
msgstr "LDAP 不支援變更名稱"

#: keystone/identity/backends/ldap/core.py:907
#, fuzzy, python-format
msgid "User %(user_id)s is already a member of group %(group_id)s"
msgstr "使用者 %s 已是群組 %s 的成員"

#: keystone/identity/backends/ldap/core.py:950
#, python-format
msgid ""
"Group member '%(user_dn)s' not found in '%(group_dn)s'. The user should "
"be removed from the group. The user will be ignored."
msgstr ""

#: keystone/openstack/common/policy.py:394
#, python-format
msgid "Failed to understand rule %(rule)s"
msgstr "無法理解規則 %(rule)s"

#: keystone/openstack/common/policy.py:404
#, python-format
msgid "No handler for matches of kind %s"
msgstr "類型為 %s 的相符項沒有處理程式"

#: keystone/openstack/common/policy.py:679
#, python-format
msgid "Failed to understand rule %(rule)r"
msgstr "無法理解規則 %(rule)r"

#: keystone/policy/backends/rules.py:93
#, python-format
msgid "enforce %(action)s: %(credentials)s"
msgstr ""

#: keystone/token/controllers.py:465 keystone/token/controllers.py:468
msgid "Token does not belong to specified tenant."
msgstr "記號不屬於所指定的 Tenant。"

#: keystone/token/controllers.py:475
msgid "Non-default domain is not supported"
msgstr "不支援非預設網域"

#: keystone/token/controllers.py:483
msgid "Domain scoped token is not supported"
msgstr "不支援網域範圍的記號"

#: keystone/token/backends/memcache.py:144
#, python-format
msgid ""
"Successful set of token-index-list for user-key \"%(user_key)s\", "
"#%(count)d records"
msgstr ""

#: keystone/token/backends/memcache.py:154
#, python-format
msgid ""
"Failed to set token-index-list for user-key \"%(user_key)s\". Attempt "
"%(cas_retry)d of %(cas_retry_max)d"
msgstr ""

#: keystone/token/backends/memcache.py:163
msgid "Unable to add token user list"
msgstr ""

#: keystone/token/backends/memcache.py:172
msgid "Unable to add token to revocation list."
msgstr "無法將記號新增至撤銷清冊。"

#~ msgid "HTTP PERF: %(time).5f seconds to %(method)s %(host)s:%(port)s %(path)s)"
#~ msgstr "HTTP PERF：%(time).5f 秒鐘以 (%(method)s %(host)s:%(port)s %(path)s)"

#~ msgid "Unable to add token user list."
#~ msgstr "無法新增記號使用者清單。"

